brainscore/benchmarks/hebart2023.py

```diff
@@ -19,20 +20,33 @@ def __init__(self):
         self._visual_degrees = None
 
         self._number_of_trials = None
+        
+        #TODO I am not sure where this goes , but it should be in the benchmark right?
+        self.validation_data = None  # https://osf.io/b2a4j we'll need to convert this to DataAssembly right?
 
-        super(_Hebart2023Accuracy, self).__init__(
+        # TODO do we put the similarity matrix inside look_at?
```

Comment:
if you mean the human data, that is just kept in the assembly and then compared against the model assembly in `__call__` and more specifically the metric